src/components/CreateAnalysis/CreateAnalysisWrapper.tsx

```diff
@@ -14,18 +14,25 @@ const CreateAnalysisWrapper = () => {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const history = useHistory();
 
-  const submitAnalysis = () => {
-    const imagesSelected: DcmImage[] = CreateAnalysisService.pickImages(dcmImages, selectedStudyUIDs);
-    if (imagesSelected.length <= 0) {
-      setIsModalOpen(true);
-      return;
+  const submitAnalysis = (XrayModel: string | void, CTModel: string | void) => {
+    return () => {
+      const imagesSelected: DcmImage[] = CreateAnalysisService.pickImages(dcmImages, selectedStudyUIDs);
+      if (imagesSelected.length <= 0) {
+        setIsModalOpen(true);
+        return;
+      }
+      // Updating staging images
+      dispatch({
+        type: StagingDcmImagesTypes.UpdateStaging,
+        payload: { imgs: imagesSelected }
+      })
+      history.push({ // Passing route parameters between components to determine which models are to be used for the analysis
+        pathname: "/",
+        state: { 
+          XrayModel: XrayModel,
+          CTModel: CTModel
+        }});
```

Comment:
Yeah, was originally thinking same thing, but from talking to Boris, realized that it may be better not to pollute the global scope with using Context (since this information is only being used in one place)

Reply:
My original comment was about being careful with what we put into Context store in order not to bloat the store.

@colinywu I might be wrong, but it seems like `XrayModel` and `CTModel` are passed through the components up and down relatively often. Can you confirm this? In this case, I do suggest keeping it in the store. However, given that we are still working with mocked data, maybe this can be something we can change later (location state -> context).